10_rest/n0npax/cmd/puppy-server/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+
+	puppy "github.com/anz-bank/go-course/10_rest/n0npax/pkg/puppy"
+	store "github.com/anz-bank/go-course/10_rest/n0npax/pkg/puppy/store"
+	"github.com/gin-gonic/gin"
+)
+
+var (
+	out io.Writer = os.Stderr
+)
+
+func main() {
+	config, err := parseArgs()
+	puppyFilePath, storeType, port := config.path, config.sType, config.port
+	exitOnError(err)
+
+	storer, err := createStorer(storeType)
```

Comment:
imo `config.storeType` would be better here.

Reply:
fixed